specification/edgeorder/resource-manager/Microsoft.EdgeOrder/2020-12-01-preview/examples/AddressDelete.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "resourceGroupName": "TestRG",
+    "addressName": "TestAddressName1",
+    "api-version": "2020-12-01-preview",
+    "User-Agent": [
+      "vscode-restclient"
+    ],
+    "Referer": [
+      "https://localhost/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourceGroups/TestRG/providers/Microsoft.EdgeOrder/addresses/TestAddressName1?api-version=2020-12-01-preview"
+    ]
+  },
+  "responses": {
+    "200": {
```

Comment:
>200 [](start = 5, length = 3)

Will you check if there are open order with this address referred?

Reply:
No we are not checking it